README.md

```diff
@@ -62,6 +62,10 @@ How to [secure your setup](/doc/security/ssl.md).
 - Electron and ChromeOS applications to bridge the gap between local<->remote.
 - Run VS Code unit tests against our builds to ensure features work as expected.
 
+### Notes
+
+- VS Code Marketplace cannot be used, we're not allowed to use it. However, the open source extensions should all be allowed to work under `code-server` just fine.
```

Comment:
My recommendations for the change here would be:

`At the moment we can't use the official VSCode Marketplace. Coder has created a collection of extensions that are open-source. While you can't directly download extensions from the official VSCode Marketplace, you can upload them to be installed in `code-server`.`